bot/tests/administration/test_report_issue.py

```diff
@@ -10,32 +12,50 @@ class TestReportCommand(BaseTest):
     @pytest.mark.quick
     @pytest.mark.asyncio
     async def test_report_with_valid_message(self):
-        await self.expect_command_result_contains(
-            '/report To i to nie działa',
-            [msg.get_report_received_message()],
-        )
+        message = 'To i to nie działa'
+        await self.expect_command_result_contains(f'/report {message}', [msg.get_report_received_message()])
 
     @pytest.mark.quick
     @pytest.mark.asyncio
     async def test_report_with_empty_message(self):
-        await self.expect_command_result_contains(
-            '/report',
-            [msg.get_no_report_content_message()],
-        )
+        command = '/report'
+        await self.expect_command_result_contains(command, [msg.get_no_report_content_message()])
 
-    @pytest.mark.long
+    @pytest.mark.quick
     @pytest.mark.asyncio
     async def test_report_with_special_characters_in_message(self):
-        await self.expect_command_result_contains(
-            '/report To się nie działa @#$%^&*()!',
-            [msg.get_report_received_message()],
-        )
+        message = 'To się nie działa @#$%^&*()!'
+        await self.expect_command_result_contains(f'/report {message}', [msg.get_report_received_message()])
+
+    # @pytest.mark.quick
+    # @pytest.mark.asyncio
+    # async def test_report_with_long_message(self):
+    #     long_message = "To i to nie działa. " * 4000
+    #     await self.expect_command_result_contains(
+    #         f'/report {long_message}',
+    #         [msg.get_limit_exceeded_report_length_message()]
+    #     )
+
+    @pytest.mark.quick
+    @pytest.mark.asyncio
+    async def test_report_with_alias_command(self):
+        message = 'Alias działa poprawnie'
+        await self.expect_command_result_contains(f'/r {message}', [msg.get_report_received_message()])
+
+    @pytest.mark.quick
+    @pytest.mark.asyncio
+    async def test_report_with_multiple_spaces(self):
+        message = '   To się nie działa   '
+        await self.expect_command_result_contains(f'/report {message}', [msg.get_report_received_message()])
 
-    @pytest.mark.long
+    @pytest.mark.quick
     @pytest.mark.asyncio
-    async def test_report_with_long_message(self):
-        long_message = "To i to nie działa. " * 20
-        await self.expect_command_result_contains(
-            f'/report {long_message}',
-            [msg.get_report_received_message()],
-        )
+    async def test_report_logs(self):
+        message = 'To i to nie działa'
+
+        await self.expect_command_result_contains(f'/report {message}', [msg.get_report_received_message()])
+
+        reports = await DatabaseManager.get_reports(s.DEFAULT_ADMIN)
+
+        assert len(reports) > 0, "Zgłoszenie nie zostało zapisane w bazie danych."
+        assert reports[0]['report'] == message, f"Oczekiwano: {message}, otrzymano: {reports[0]['report']}"
```

Comment:
zażądaj zwrotu pieniędzy od openai xD czemu z uporem maniaka forsuje na tobie te pierdolone asserty zamiast korzystać z biblioteki testowej?

Reply:
i czemu to jest po polskiemu xD